plugin/docker/src/main/java/com/perl5/lang/perl/idea/sdk/host/docker/PerlDockerAdapter.java

```diff
@@ -110,6 +110,30 @@ public PerlDockerAdapter(@NotNull PerlDockerData data) {
     return containerName;
   }
 
+  private boolean tryCopyWithTar(@NotNull String containerName, @NotNull String remotePath, @NotNull String localPath)
+    throws ExecutionException {
+    File localPathFile = new File(localPath);
+    File remotePathFile = new File(remotePath);
+    String command = String.join(" ", DOCKER_EXECUTABLE, EXEC, INTERACTIVELY, containerName,
+                                 "tar", "hCcF", remotePathFile.getParent(), "-", remotePathFile.getName(),
+                                 "|", "tar", "Cxf", localPathFile.getParent(), "-");
+    if (SystemInfo.isLinux) {
+      try {
+        checkOutput(PerlHostData.execAndGetOutput(
+          new PerlCommandLine("sh").withHostData(PerlHostHandler.getDefaultHandler().createData())
```

Comment:
I suppose there is a method to run command in docker. So you shouldn't do this low-level running yourself.

Reply:
Here is another option do it via mount and copy but we can mess with file access right that way

Reply:
current method do this as well. There is a `com.perl5.lang.perl.idea.sdk.host.PerlHostData#fixPermissionsRecursively` to handle this